servicenow-instance-wakeup.go

```diff
@@ -150,8 +150,19 @@ func wakeUpInstance(ctx context.Context, username string, password string, timeo
 		fmt.Printf("Login successful!\n")
 	}
 
+    fmt.Printf("Wait button")
+    chromedp.WaitVisible(`document.querySelector("body > dps-app").shadowRoot.querySelector("div > header > dps-navigation-header").shadowRoot.querySelector("header > div > div.dps-navigation-header-utility > ul > li.dps-navigation-header-list-item > dps-login").shadowRoot.querySelector("div > button")`, chromedp.ByJSPath)
+    fmt.Printf("Button is it")
+
+    if err := chromedp.Click(`[@class='dps-login']`, chromedp.BySearch); err != nil {
```

Comment:
Давай спробуємо так:
```suggestion
    profileSelector := `document.querySelector("body > dps-app").shadowRoot.querySelector("div > header > dps-navigation-header").shadowRoot.querySelector("header > div > div.dps-navigation-header-utility > ul > li.dps-navigation-header-list-item > dps-login").shadowRoot.querySelector("div > button")`
    chromedp.WaitVisible(profileSelector, chromedp.ByJSPath)
    fmt.Printf("Button is it")

    if err := chromedp.Click(profileSelector, chromedp.BySearch); err != nil {
```

Reply:
а, стоп